source/dev/idea_list.md

```diff
@@ -0,0 +1,147 @@
+# Google Summer of Code 2021
+Breakthrough Energy Sciences (BES) has applied to serve as a mentoring organization for Google Summer of Code (GSoC).
+The GSoC program offers each student a stipend to work on open source code during the summer. 
+If you are interested in the program [here](https://summerofcode.withgoogle.com/) you can find more information.
+
+If you are a student and new to GSoC, please read the following [guide](https://google.github.io/gsocguides/student/).
+If you are new to BES, look [here](https://science.breakthroughenergy.org/) to learn more about our organization.
+
+
+## Community and Communication
+We are active on Slack and this is the best way to reach us.
+[Join us here](http://science.breakthroughenergy.org/#get-updates), introduce yourself, discuss your potential project in the #ideas channel, and let us know if you have any questions.
+
+If you need to reach us via email, here you go: sciences@breakthroughenergy.org
+
+Please review our [code of conduct](code_of_conduct.md) before joining the community.
+## Application
+If you are a student and applying for GSoC find bellow the idea list and explore the projects we would like to accomplish with you this summer.
```

Comment:
If you are a student and applying for GSoC find below the idea list --> If you are a student and applying for GSoC you will find below a list of projects we would like to accomplish with you this summer